src/popup/router/components/TransactionOverview.vue

```diff
@@ -0,0 +1,272 @@
+<template>
+  <div class="transaction-overview">
+    <div class="title">
+      {{ transaction.title }}
+      <span class="description">({{ transaction.description }})</span>
+    </div>
+
+    <div class="parties">
+      <Avatar v-bind="transaction.sender" />
+      <div class="mid">
+        <TriangleRight />
+        <div class="line" />
+      </div>
+      <div v-if="transaction.aens" class="avatar">
+        <AensIcon class="icon" />
+      </div>
+      <div v-else-if="transaction.contractCreate" class="avatar">
+        <ActionIcon class="icon" />
+      </div>
```

Comment:
```suggestion
```
this doesn't seem to be needed because it would be handled the same by the last `v-else`